app/views/users/show_follow.html.erb

```diff
@@ -0,0 +1,31 @@
+<%provide(:title), @title%>
```

Comment:
I am having this error.
![Screen Shot 2021-05-03 at 11 17 00 PM](https://user-images.githubusercontent.com/74635797/116895435-ca431380-ac65-11eb-9e47-cc1ab23b6532.png)

Place the **@title** inside the parenthesis.
`<% provide(:title, @title) %>`